src/EntryPoints/Specials/SpecialManageApprovers.php

```diff
@@ -84,6 +85,10 @@ private function renderHtml( array $approversCategories ): void {
 		$template = file_get_contents( __DIR__ . '/../../../templates/ManageApprovers.mustache' );
 		$compiledTemplate = LightnCandy::compile( $template, [ 'flags' => LightnCandy::FLAG_MUSTACHE ] );
 		$this->getOutput()->addHTML(
+			Html::rawElement( 'p',
```

Comment:
Do we allow HTML in the wikitext? If we do then it might be better to use `<div>` element instead since some HTML tags are not allowed in `<p>`

Reply:
I already merged https://github.com/ProfessionalWiki/Rules/pull/64, but I agree it might be better to use `<div>`. `parse()` is going to generate HTML from wikitext, so somebody can end up having paragraphs inside this `<p>`.

Reply:
div ends up with insufficient space (padding) below the text

Reply:
Div
<img width="470" height="130" alt="image" src="https://github.com/user-attachments/assets/b4eb2b52-7fda-42ba-8546-18982e48fc78" />

p
<img width="470" height="130" alt="image" src="https://github.com/user-attachments/assets/2c117475-e89a-4b38-a80e-acdb310bd030" />